docs/using-the-aws-driver/OktaAuthentication.md

```diff
@@ -0,0 +1,51 @@
+| `USERNAME`                    |   Yes    | The Username must be set to the [IAM database user](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html).                                                                       | `null`                   | `us-east-2`                                            |
+| `IDP_PORT`                    |    No    | The port that the host for the authentication service listens at.                                                                                                                                              | `urn:amazon:webservices` | `urn:amazon:webservices`                               |
+| `IAM_HOST`                    |    NO    | Overrides the host used to generate the authentication token. This is useful when you are connecting using a custom endpoint, since authentication tokens need to be generated using the RDS/Aurora endpoints. | `NULL`                   | `database.cluster-hash.region.rds.amazonaws.com`       |
+| `IAM_DEFAULT_PORT`            |    No    | This property overrides the default port that is used to generate the authentication token. The default port is the default MySQL port.                                                                        | `3306`                   | `1234`                                                 |
+| `IAM_TOKEN_EXPIRATION`        |    No    | Overrides the default IAM token cache expiration in seconds.                                                                                                                                                   | `900`                    | `123`                                                  |
+| `HTTP_CLIENT_SOCKET_TIMEOUT`  |    No    | The socket timeout value in milliseconds for the HttpClient used during the Okta authentication workflow.                                                                                                      | `60000`                  | `60000`                                                |
+| `HTTP_CLIENT_CONNECT_TIMEOUT` |    No    | The connect timeout value in milliseconds for the HttpClient used during the Okta authentication workflow.                                                                                                     | `60000`                  | `60000`                                                |
+| `SSL_INSECURE`                |    No    | Indicates whether or not the SSL connection is secure or not. If not, it will allow SSL connections to be made without validating the server's certificates.                                                   | `true`                   | `false`                                                |
```

Comment:
1. I don't recall seeing SSL being added for Okta specifically, perhaps this belongs somewhere else in the connection string / another document?
2. Think the wording on description is a bit misleading. Rather than saying it
`Indicates whether or not the SSL connection is **secure** or not`
would it be better to say 
`Indicates whether or not the SSL connection is **verified** or not`
Or something closer to
`Indicates whether the connection can be insecure or not`
3. Is this correct? By default the driver will be **insecure**? `SSL_INSECURE = TRUE`

Reply:
1. Yep putting a separate PR adding those params